data-scripts/lists.ts

```diff
@@ -225,6 +225,24 @@ export default {
       },
     },
   },
+  'ja': {
+    commonWords: {
+      // fixme We must convert Japanese characters (Hiragana, Katanaka and Kanji) into Romaji as a dictionary
+      // because Japanese characters are not permitted to use for password in general.
+      // It would be too difficult to do programmatically.
+      source:
+        'https://github.com/hermitdave/FrequencyWords/raw/master/content/2018/ja/ja_full.txt',
+      options: { hasOccurrences: true },
+    },
+    firstnames: {
+      source:
+        'https://raw.githubusercontent.com/tomoyukikashiro/zxcvbn-japanese-data/main/data/first-names.txt.txt',
+    },
+    lastnames: {
+      source:
+        'https://raw.githubusercontent.com/tomoyukikashiro/zxcvbn-japanese-data/main/data/last-names.txt.txt',
+    },
```

Comment:
Do you have official sources for the names, which can be downloaded and parsed?
It would be nicer but not mandatory.

Reply:
I could find an official resource for "common names" in japan.
The only one I found was [willnet/gimei](https://github.com/willnet/gimei) which was name data for test but it contains a lot of names but it only had "kanji" and "Hiragana" characters.

[As I mention](https://github.com/zxcvbn-ts/zxcvbn/pull/119#discussion_r913350845), "Kanji" and "Hiragana" are not used in password field so I converted it into "Romaji" in my repository.